Formula/mavsdk.rb

```diff
@@ -31,7 +35,53 @@ class Mavsdk < Formula
 
   uses_from_macos "zlib"
 
+  resource "Jinja2" do
+    url "https://files.pythonhosted.org/packages/7a/0c/23cbcf515b5394e9f59a3e6629f26e1142b92d474ee0725a26aa5a3bcf76/Jinja2-3.0.0.tar.gz"
+    sha256 "ea8d7dd814ce9df6de6a761ec7f1cac98afe305b8cdc4aaae4e114b8d8ce24c5"
+  end
+
+  resource "MarkupSafe" do
+    url "https://files.pythonhosted.org/packages/67/6a/5b3ed5c122e20c33d2562df06faf895a6b91b0a6b96a4626440ffe1d5c8e/MarkupSafe-2.0.0.tar.gz"
+    sha256 "4fae0677f712ee090721d8b17f412f1cbceefbf0dc180fe91bab3232f38b4527"
+  end
+
+  resource "protoc-gen-mavsdk" do
+    url "https://files.pythonhosted.org/packages/da/91/96d4bde10cf9512250290880373fec0e1e068a9cafae96e69e90f45b2616/protoc_gen_mavsdk-1.0.1-py3-none-any.whl"
+    sha256 "5ccdfadea69e2da95567a8aa1925bdb2d34697def2ce1d5835b596c97abcc1ac"
+  end
+
   def install
+    generator = "tools/generate_from_protos.sh"
+    inreplace generator do |s|
+      s.gsub!(/^(protoc_binary)=.*/, "\\1=#{Formula["protobuf"].opt_bin}/protoc")
+      s.gsub!(/^(protoc_grpc_binary)=.*/, "\\1=#{Formula["grpc"].opt_bin}/grpc_cpp_plugin")
+    end
+
+    # We need `protoc-gen-mavsdk` to bootstrap
+    venv_dir = buildpath/"bootstrap"
+    protoc_gen_dir = buildpath/"pgm"
+    site_packages = Language::Python.site_packages("python3")
+    venv = virtualenv_create(venv_dir, "python3")
+    venv.pip_install %w[Jinja2 MarkupSafe]
+    with_env(
+      VIRTUAL_ENV: venv_dir,
+      PYTHONPATH:  "#{venv_dir}:#{protoc_gen_dir}:#{Formula["six"].opt_prefix/site_packages}",
+      PATH:        "#{venv_dir}/bin:#{protoc_gen_dir}/bin:#{ENV["PATH"]}",
+    ) do
+      buildpath.install resource("protoc-gen-mavsdk")
+      system "python", "-m", "pip", "install", "-v",
+             "--no-deps",
+             "--no-index",
+             "--upgrade",
+             "--isolated",
+             "--target=#{protoc_gen_dir}",
```

Comment:
```suggestion
```
Can just remove this to install into default virtualenv directories (bin and site-packages)

Reply:
Nice. Let me check that the generator script still runs.

Reply:
Thanks, this works.